olive/passes/onnx/conversion.py

```diff
@@ -160,6 +160,7 @@ def _export_pytorch_model(
         device: Union[str, torch.device],
         torch_dtype: Optional[torch.dtype] = None,
         tempdir: Optional[Union[Path, str]] = None,
+        dynamic: bool = True,
```

Comment:
if you think the option to turn dynamic on and off is useful, I think we can make it into a config option and add it to the `_default_config` at line 50. This way we don't have to depend on model_attributes and it is instead directly provided to the pass config through `config["dynamic"]`